pints/__init__.py

```diff
@@ -80,3 +80,9 @@ def version(formatted=False):
 from _mcmc import MCMC
 from _mcmc._adaptive import AdaptiveCovarianceMCMC, adaptive_covariance_mcmc
 
+#
+# Basic plots
+#
+import _plot as plot
```

Comment:
Hi @chonlei ! This makes matplotlib importing part of loading Pints. We don't want that as servers etc. might not be able to `import matplotlib`. Could you change this to make `_plot.py` into `plot.py` (so an official submodule)? Users will then have to do things like
```
import pints
import pints.plot
...
pints.plot.trace(chain)
```

Reply:
Done